rpcserver.go

```diff
@@ -1211,7 +1211,7 @@ out:
 func (r *rpcServer) AbandonChannel(ctx context.Context,
 	in *lnrpc.AbandonChannelRequest) (*lnrpc.AbandonChannelResponse, error) {
 
-	if !DebugBuild {
+	if build.Deployment != build.Development {
```

Comment:
nit: I imagine this could be a bit confusing for someone reading it the first time (comparing two constants, it is not obvious that they are build flag gated). Maybe update the `godoc` for `build.Deployment` to say something like "Deployment is set to the deployment type of the current build. For dev builds this will be Development".

Reply:
added `IsDevBuild()` and `IsProdBuild()` helpers that are now exported by `build`